### PR TITLE
test changes to tool tips up for review 

### PR DESCRIPTION
**Pull request by Snoopy360**
removed spell amp value line from athers lense
added more thorough description of drums behavior to upgraded drums of endurance
removed structure info from assault curass
 removed rosh disable info from blink
removed heals allies on death info from blood stone
added lore line into the infinity bottle (coders need to check this for bugs)

### CODE REVIEW

File: game/resource/English/ability/items/tooltip_ancient_janggo.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 "DOTA_Tooltip_Ability_item_ancient_janggo_2"            "Drum of Endurance"
 "DOTA_Tooltip_Ability_item_recipe_ancient_janggo_2"     "Drum of Endurance Recipe"
-"DOTA_Tooltip_Ability_item_ancient_janggo_2_Description"                        "<h1>Active: Endurance</h1> Gives +%bonus_attack_speed_pct% attack speed and +%bonus_movement_speed_pct%%% movement speed to nearby allies for %duration% seconds. <br><br>Radius: %radius%\n<h1>Passive: Swiftness Aura</h1>Gives %bonus_aura_movement_speed% movement speed to nearby allies. <br><br>Radius: %radius%"
```
Comment (imaGecko) [inline game/resource/English/ability/items/tooltip_ancient_janggo.txt]: drums is missing the change in the t1 item, arguagbly not needed anywhere but t1

File: game/resource/English/ability/items/tooltip_infinity_bottle.txt [hunk 2]
```diff
@@ -3,5 +3,6 @@
 //=============================================================================
 "DOTA_Tooltip_Ability_item_infinite_bottle"                                         "Infinity Bottle"
 "DOTA_Tooltip_Ability_item_infinite_bottle_Description"                             "<h1>Active: Regenerate</h1>Consumes a charge to restore %health_restore% health and %mana_restore% mana over %restore_time% seconds.<br><br>Initial Charges: 3<br>Max Charges: Infinite"
+"DOTA_Tooltip_ability_item_bottle_Lore"                                             "in an attempt to destroy our world the Whitelord gave nutrals all bottles. As the flood of enchantments began crashing games the Darklord came and told the heroes "bottles now stack!" thus the infintiy bottle was born"
```
Comment (imaGecko) [inline game/resource/English/ability/items/tooltip_infinity_bottle.txt]: Functional issues: You used "" as part of the tooltip, so your lore ends after heroes. unexpected token of bottles now stack and then a key of thus the infinity bottle was born.
Also it's the lore for vanilla bottle which you can no longer buy or drop.

**Comment by Snoopy360** [comment]
fixed drums and bottle issues

**Comment by chrisinajar** [comment]
This branch is a mess and you have 3 PR's open with a bunch of duplicate work across em. Can you clean things up?

**Pull request closed without merging**